js/CurrentPeriodLineItemHandler.js

```diff
@@ -693,6 +688,22 @@ CRM.RecurringContribution.CurrentPeriodLineItemHandler = (function($) {
       var taxRate = Math.round(taxAccount.tax_rate * 100) / 100;
       CRM.$('#newline_tax_rate', this.newMembershipRow).html(taxRate + ' %');
     }
+
+    var params = this.buildNextPeriodLineItemCallParameters();

```

Comment:
also update buildNextPeriodLineItemCallParameters() to have limit => 0 to fetch all line items